modules/ebs_snapshot_explorer/__init__.py

```diff
@@ -0,0 +1,34 @@
+import argparse
```

Comment:
Is this supposed to be in __init__.py? If we are going to make changes like this, we should standardize them if possible